root/layout/components/sidebar/PlayOnListenBrainzButton.js

```diff
@@ -11,19 +11,28 @@ import listenBrainzIconUrl
   from '../../../static/images/icons/listenbrainz.png';
 
 function getListenBrainzLink(
-  entityType: 'release' | 'recording',
+  entityType: 'album' | 'artist' | 'recording',
   mbids: string | $ReadOnlyArray<string>,
 ): string | null {
   let formattedMbids;
 
   if (Array.isArray(mbids)) {
-    formattedMbids = mbids.join(',');
+    if (entityType === 'recording') {
+      formattedMbids = encodeURIComponent(mbids.join(','));
```

Comment:
The request to LB works regardless of whether the `,` is encoded. (Just thought I'd mention, since I wasn't sure if it would.)